.vscode/launch.json

```diff
@@ -2,8 +2,8 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "command": "yarn test -i --watchAll",
-      "name": "Debug yarn test",
+      "command": "yarn react-start",
```

Comment:
Because this will run only the react process and won't open electron

Reply:
there is another task below, see 'Debug Main Process'. I already tested it.